README.md

```diff
@@ -5,6 +5,8 @@
 VCF-formatted files are the lingua franca of next-generation sequencing, whereas [HL7 FHIR](https://www.hl7.org/fhir/) is emerging as a standard language for electronic health record interoperability. A growing number of clinical genomics applications are emerging, based on the [HL7 FHIR Genomics standard](http://hl7.org/fhir/uv/genomics-reporting/index.html). Here, we provide an open source utility for converting variants from VCF format into HL7 FHIR Genomics format. Details of the translation logic are on the [manual page](docs/Manual.md). Additional information and case studies are described in our [BMC Bioinformatics article](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04039-1). Conceptually, the utility takes a VCF as input and outputs a [FHIR Genomics report](http://hl7.org/fhir/uv/genomics-reporting/index.html). 
 
 ### Install
+Install and add [Ubuntu WSL](https://www.microsoft.com/en-in/p/ubuntu-1804-lts/9n9tngvndl3q?rtc=1&activetab=pivot:overviewtab) to the [windows terminal](https://www.microsoft.com/en-in/p/windows-terminal/9n0dx20hk701?activetab=pivot:overviewtab). (Only for windows users!)
+
```

Comment:
#### Pre-requisite

##### Windows

* vcf2fhir is supported only on Unix and Linux systems. To run it on windows install [Ubuntu WSL](https://www.microsoft.com/en-in/p/ubuntu-1804-lts/9n9tngvndl3q?rtc=1&activetab=pivot:overviewtab).

#### vcf2hfir

Reply:
Do we need: add to the [windows terminal](https://www.microsoft.com/en-in/p/windows-terminal/9n0dx20hk701?activetab=pivot:overviewtab) in the document? Are there special steps required to add it to the terminal? If yes, we should list that instead of just a statement.

Reply:
I also wanted to investigate why does user needs to install `pip install cython wheel`, but I never got enough time. Can you help look at this too?  
I think we are missing something in our build process. `cython` and `wheel` should only be needed for running from the source. If people are downloading from `pip` they should get the compiled binary.

Reply:
Hi @srgothi92,

> Do we need: add to the windows terminal in the document? Are there special steps required to add it to the terminal? If yes, we should list that instead of just a statement.

Should I add this [link](https://windowsloop.com/add-ubuntu-to-windows-terminal/) for the steps to add Ubuntu WSL to the windows terminal?

> why does user needs to install pip install cython wheel

I checked this one, and I think it is a requirement for [pyrle](https://github.com/pyranges/pyrle). When I tried installing vcf2fhir in a virtual environment without installing cython wheel I received the following error:

![Screenshot 2021-06-13 at 9 58 30 PM](https://user-images.githubusercontent.com/47586886/121815144-88f54900-cc92-11eb-8b0c-1ff81b46f3c6.jpg)

Reply:
> Should I add this link for the steps to add Ubuntu WSL to the windows terminal?

Without these steps, you can't use WSL? 

> I checked this one, and I think it is a requirement for pyrle. 

That's strange. What happens if you try to `pip install pyranges` do we get the same error?  If not, then it's strange that we get this error. If we do get an errr, we should check with pyranges developer.

Reply:
> Without these steps, you can't use WSL?

We can use Ubuntu WSL without adding it to the windows terminal, but the friend of mine whom I asked about it, told me that he faced a lot of issues when directly using it, and this is the easiest way to work with a WSL.

 > What happens if you try to pip install pyranges do we get the same error?

Yes, we get the same error because it too has `pyrle` as one of its requirements. We get the error because, [here](https://github.com/pyranges/pyrle/blob/master/setup.py#L7) in the `setup.py` file for `pyrle`, `Cython` is imported. Therefore, it needs to be installed before installing `pyrle` or a package with `pyrle` as a requirement.

Reply:
> Yes, we get the same error because it too has pyrle as one of its requirements

Can we open an issue on `pyranges` and see what the maintainer has to say.

Reply:
> We can use Ubuntu WSL without adding it to the windows terminal, but the friend of mine whom I asked about it, told me that he faced a lot of issues when directly using it, and this is the easiest way to work with a WSL.

I think I would have to try WSL on my system. The steps mentioned in the blog you shared are too huge to put in our docs.

Reply:
> Can we open an issue on pyranges and see what the maintainer has to say.

I just looked at the open issues in `pyranges` and they seem to be working on this one([here](https://github.com/biocore-ntnu/pyranges/issues/85)).  In that issue, there is a mention of this [pull request](https://github.com/pyranges/sorted_nearest/pull/4) in `sorted_nearest`. That pull request adds the build-time requirements for the package. Should we similarly add `setuptools`, `Cython`, and `Wheel` as build-time requirements for `vcf2fhir`?

Reply:
> That pull request adds the build-time requirements for the package. Should we similarly add setuptools, Cython, and Wheel as build-time requirements for vcf2fhir or wait for pyranges to be updated?

Sure, you can try that if it works and open an issue in our repository to fix it later when it is addressed upstream.

Reply:
Okay I was finally able to find some time and install `vcf2fhir` on my windows machine using WSL. However, with this issue I was looking to install `vcf2fhir` natively on windows. I tried `pysam` and it install fine on windows with conda packaging. May be we should try conda package on windows instead of suggesting WSL path.

Reply:
> May be we should try conda package on windows instead of suggesting WSL path.

Sure, I will try installing `vcf2fhir` via `conda` once it gets **deployed** on the `bioconda` channel. :)

Reply:
> Sure, I will try installing vcf2fhir via conda once it gets deployed on the bioconda channel. :)

Before publishing to `biconda` channel we should first test with out personal channel similar to what we do with test pypi.